doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs.rst

```diff
@@ -0,0 +1,271 @@
+  Included support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+
+Security libraries
+------------------
+
+* :ref:`nrf_security` library:
+
+  * Added support for PSA crypto on the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>`.
+    Enabled the PSA crypto support with a CRACEN driver as the backend by setting the Kconfig option :kconfig:option:`CONFIG_NRF_SECURITY`.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+|no_changes_yet_note|
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``0051731a41fa2c9057f360dc9b819e47b2484be5``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline 0051731a41 ^23cf38934c
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^0051731a41
+
+The current |NCS| main branch is based on revision ``0051731a41`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Zephyr samples:
+
+* Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` in the following Zephyr samples:
+
+  * :ref:`zephyr:blinky`
+  * :ref:`zephyr:hello_world`
+  * :ref:`zephyr:settings`
+  * :ref:`zephyr:bluetooth-observer-sample`
+  * :ref:`zephyr:nrf-system-off-sample`
+  * :ref:`zephyr:peripheral_hr`
+  * :ref:`zephyr:bluetooth_central_hr`
+  * :ref:`zephyr:smp-svr`
+
+Current nRF54L15 PDK Limitations
+================================
+
+General
+-------
+
+* The v0.2.1 revision of the nRF54L15 PDK has Button 3 and Button 4 connected to GPIO port 2 that does not support interrupts.
+  The workaround for this issue is enabled by default with the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option, but it increases the overall power consumption of the system.
```

Comment:
```suggestion
  An example of software workaround for this issue was implemented in :ref:`dk_buttons_and_leds_readme` library.
  The workaround is enabled by default with the :kconfig:option:`CONFIG_DK_LIBRARY_BUTTON_NO_ISR` Kconfig option, but it increases the overall power consumption of the system.
```

Maybe we could be a bit more specific here and inform that the workaround is part of one of our libraries (it would not help if user uses e.g. GPIO driver from Zephyr)

Reply:
good stuff, thanks